### DISCUSSION TITLE
Actions according to the Gazebo frame

### DISCUSSION THREAD

**Discussion by Kai-X-Org** [HumanSupportRobot/QandA#67] [Q&A]
We see that the interface can conduct actions according to the 'map', 'base', etc. Is it possible to let it conduct actions according to the Gazebo native frame? Thanks!

**Comment by masudam** [comment]
I think there is no way to use Gazebo frames directly. However, if you can convert Gazebo frames to tf and publish them, you should be able to create actions that utilize the tf data.
